src/github/middleware.ts

```diff
@@ -81,7 +81,6 @@ export default (
         {
           noop: 'bot',
           botId: context.payload.sender.id,
-          botLogin: context.payload.sender.login,
```

Comment:
I was unable to reproduce this log, however `botLogin` doesn't sound like something we can log